src/Lorentz/Contracts/BaseDAO/TZIP16Metadata.hs

```diff
@@ -0,0 +1,46 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+-- | TZIP-16 metadata.
+module Lorentz.Contracts.BaseDAO.TZIP16Metadata
+  ( knownBaseDAOMetadata
+  ) where
+
+import Lorentz hiding (View)
+import Universum (one)
+
+import Lorentz.Contracts.Spec.TZIP16Interface
+
+import Lorentz.Contracts.BaseDAO.Types
+
+-- For metadata storage does not matter
+type MetadataStorage = Storage () ()
+
+-- | Parts of metadata that can be filled just knowing the contract,
+-- without user's input.
+knownBaseDAOMetadata :: Metadata (ToT MetadataStorage)
+knownBaseDAOMetadata = mconcat
+  [ version "0.0.1"
+  , license $ License "LicenseRef-MIT-TQ" Nothing
+  , authors [Author "Serokell", Author "Tocqueville Group"]
+  , homepage "https://github.com/tqtezos/baseDAO"
+  , interfaces [tzip 12, tzip 16, tzip 17]
```

Comment:
AFAIK `tzip 16` shouldn't be mentioned because it's redundant (one should get `%metadata` before accessing `interfaces` and it already implies that the contract follows TZIP-16).

Reply:
Btw, examples in TZIP-16 mention TZIP-16 as one of implemented interfaces:
https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-16/examples/example-002.json

Though indeed this does not mean we have to follow this.

Reply:
They say: "Each string should allow the consumer of the metadata to know which interfaces and behaviors the contract claims to obey (other than the obvious TZIP-016)."

I think "other than the obvious TZIP-016" means we don't have to include it. Probably it's ok to include it (not an error), but it seems useless.